source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/list/index.tsx

```diff
@@ -43,14 +43,16 @@ export default compose (
   withProjects({ all: true }),
   withLoader(({ error, projects }) => !error && !projects),
   withError('error', ({ error }) => error !== undefined),
-  withProps(({ projects }) => ({
-    projects: projects.filter(resource => resource.type === PROJECT),
-    tableName: 'directory'
-  })),
   withCache(() => ({
     organizations: q => q.findRecords(ORGANIZATION),
     groups: q => q.findRecords(GROUP),
+    projects : q => q.findRecords(PROJECT)
   })),
+  withProps(({ projects }) => ({
+    projects: projects.filter(resource => resource.type === PROJECT && resource.attributes.dateArchived == null),
```

Comment:
why was checking archived date added here?
wouldn't the backend be responsible for only sending us what we can see?
this feels like obfuscating a potential security problem

Reply:
based on my exploration here: https://github.com/sillsdev/appbuilder-portal/pull/318
I'd like to propose we not filter like this

Reply:
Agree

Reply:
I will update when you merge into master